src/main/java/nl/tudelft/scrumbledore/ScrumbledoreGUI.java

```diff
@@ -147,7 +148,83 @@ public void handle(ActionEvent arg0) {
     bottomItems.getChildren().addAll(startStopButton, settingsButton, exitButton);
     contentHandler.setBottom(bottomItems);
 
-    // Displaying the user interface.
-    gameStage.show();
+    // Calling the dynamic handling for the GUI.
+    spawnDynamic(gameStage, gamePainter);
+  }
+
+  /**
+   * spawnDynamic takes care of elements which must be updated, such as players and enemies.
+   * 
+   * @pre method called within the class and valid parameters passed
+   * @post spawns and handles dynamic elemens of the game
+   * @param passedStage
+   *          The stage used by the rest of the GUI
+   * @param passedScene
+   *          The scene used by the rest of the GUI
+   * @param passedPane
+   *          The layout pane used by the rest of the GUI
+   * 
+   */
+  private void spawnDynamic(Stage passedStage, final GraphicsContext gamePainter) {
+    new AnimationTimer() {
+      public void handle(long currentNanoTime) {
+        // Clear canvas
+        gamePainter.clearRect(0, 0, Constants.GUIX, Constants.GUIY);
+        // background image clears canvas
+        // Adding the initial player location to the GUI.
+        playerSprite = new Image(Constants.PLAYER_SPRITE);
+        gamePainter.drawImage(playerSprite,
+            game.getCurrentLevel().getPlayer().getPosition().getX(), game.getCurrentLevel()
+                .getPlayer().getPosition().getY());
+
+        // Placing the platform elements within the level.
+        for (Platform current : game.getCurrentLevel().getPlatforms()) {
+          // Painting the current platform image at the desired x and y location given by the
+          // vector.
+          gamePainter.drawImage(new Image(Constants.PLATFORM_SPRITE), current.getPosition().getX(),
+              current.getPosition().getY());
+        }
+      }
+    }.start();
+
+    passedStage.show();
+  }
+
+  private void addKeyEventListeners(Scene scene) {
+    // KeyPress Event handlers.
+    scene.setOnKeyPressed(new EventHandler<KeyEvent>() {
+
+      // Handling the key press.
+      public void handle(KeyEvent keyPressed) {
+        String keyPress = keyPressed.getCode().toString();
+
+        // Mapping the desired keys to the desired actions.
+        if (keyPress.equals("LEFT")) {
```

Comment:
One small thing, couldn't this better be done better by comparing keycodes instead of strings?